PendingReleaseNotes.md

```diff
@@ -3,5 +3,6 @@
 ## Breaking Changes
 
 - Removed support for Ceph Pacific (v16)
+- Disable webhook(if enabled) before upgrading to v1.13
```

Comment:
Two thoughts:

1. This is an instruction but not a note. I think we need to add the note first, followed by instruction: "Support for the admission controller has been removed. CRD validation is now enabled with [Validating Admission Policies](https://kubernetes.io/docs/reference/access-authn-authz/validating-admission-policy/). This requires Kubernetes version ??.??? or higher.

2. This says to disable the webhook, but it would be best if told users how to disable it as well, especially since we are removing documentation regarding the webhook. This can link to the right section in the upgrade doc for instructions.

Reply:
I have added the same intrusion in the release notes and rook upgrade doc. But instead of [VAP](https://kubernetes.io/docs/reference/access-authn-authz/validating-admission-policy/) which is beta in 1.28 I think we have to use [CEL](https://kubernetes.io/docs/reference/using-api/cel/) which is in beta 1.25.